frame/support/procedural/src/construct_runtime/expand/freeze_reason.rs

```diff
@@ -35,6 +35,7 @@ pub fn expand_outer_freeze_reason(pallet_decls: &[Pallet], scrate: &TokenStream)
 	}
 
 	quote! {
+		/// A reason for placing a freeze on funds.
```

Comment:
I think it would be better to forward the real docs put on top of the type?

Reply:
Isn't that what this is already doing? It's documenting a type that is expanded by the proc macro.

Reply:
This isn't forwarding the docs that I as a user put onto the type?

Reply:
Ahh okay, missed that this is the runtime level declaration.